rust_src/crates/webrender/src/output.rs

```diff
@@ -116,34 +133,37 @@ impl Output {
 
         renderer.set_external_image_handler(external_image_handler);
 
+        let epoch = Epoch(0);
         let pipeline_id = PipelineId(0, 0);
-        let mut txn = Transaction::new();
-        txn.set_root_pipeline(pipeline_id);
 
+        let api = sender.create_api();
         let device_size = {
             let size = window.inner_size();
             DeviceIntSize::new(size.width as i32, size.height as i32)
         };
-
-        let mut api = sender.create_api();
-
         let document_id = api.add_document(device_size);
-        api.send_transaction(document_id, txn);
```

Comment:
I believe Linux wayland platform need those codes to have a better UI startup experience.